setup.cfg

```diff
@@ -0,0 +1,12 @@
+[flake8]
+exclude =
+    .git,
+    __pycache__,
+    venv,
+    dist,
+    build
+max-line-length = 99
+
+[pycodestyle]
+max-line-length = 99
+max-doc-length= 200
```

Comment:
is this PEP8 compliant?

Reply:
How to handle long URLs in doc? 

We can strictly follow with PEP8 and disable lint on such lines manually. 
What do you think?

Reply:
using `noqa`? I think that's fine